actionController/index.js

```diff
@@ -6,6 +6,8 @@ var Connection      = require('../util/connection');
 var Routes          = require('./Routes')
 var LogModel        = require('../db_models/LogModel.js')
 
+process.title = "piGarden-AC"
```

Comment:
I used in logs just `AC`, how about dropping `piGarden-`? Is this name gonna be used by pm2?